.ci/Jenkinsfile-extension-vsce

```
@@ -106,9 +106,17 @@ pipeline {
                             else
                                 ${WORKSPACE}/env/bin/pip install inmantals=="${LS_version}"
                             fi
-                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep inmantals | cut -d'=' -f3
+                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep inmantals | cut -d'=' -f3)
```

I would make the grep more specific. Otherwise we might accidentally match an invalid line and that would be disaster if we push that into the release.

```suggestion
                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep -E '^inmantals==[0-9]+\.[0-9]+\.[0-9]+$' | cut -d'=' -f3)
```